dipy/workflows/segment.py

```diff
@@ -118,51 +117,46 @@ def run(self, streamline_files, model_bundle_files,
             The path of model bundle files
         greater_than : int, optional
             Keep streamlines that have length greater than
-            this value (default 50) in mm.
+            this value in mm.
         less_than : int, optional
             Keep streamlines have length less than this value
-            (default 1000000) in mm.
+            in mm.
         no_slr : bool, optional
             Don't enable local Streamline-based Linear
-            Registration (default False).
+            Registration.
         clust_thr : float, optional
-            MDF distance threshold for all streamlines (default 15)
+            MDF distance threshold for all streamlines
         reduction_thr : float, optional
-            Reduce search space by (mm) (default 15)
+            Reduce search space by (mm)
         reduction_distance : string, optional
-            Reduction distance type can be mdf or mam (default mdf)
+            Reduction distance type can be mdf or mam
         model_clust_thr : float, optional
-            MDF distance threshold for the model bundles (default 2.5)
+            MDF distance threshold for the model bundles
         pruning_thr : float, optional
-            Pruning after matching (default 8).
+            Pruning after matching
         pruning_distance : string, optional
-            Pruning distance type can be mdf or mam (default mdf)
+            Pruning distance type can be mdf or mam
         slr_metric : string, optional
             Options are None, symmetric, asymmetric or diagonal
-            (default symmetric).
         slr_transform : string, optional
             Transformation allowed. translation, rigid, similarity or scaling
-            (Default 'similarity').
         slr_matrix : string, optional
             Options are 'nano', 'tiny', 'small', 'medium', 'large', 'huge'
-            (default 'small')
         refine : bool, optional
-            Enable refine recognized bunle (default False)
+            Enable refine recognized bunle
```

Comment:
I think the commit can be used to fix existing typos, as well:
bunle: bundle.

Reply:
Addressed in e5af17f